### PR TITLE
the one that adds a full bleed utility class

### PR DESCRIPTION
**Pull request by sturobson**

### CODE REVIEW

**Comment by sturobson** [comment]
There's some things here I still need to look at. As 'via the twitters' `100vw` breaks a little if the browser has visible scrollbars. 

There's a couple of options:

1. 10 lines of JS that add a class to the body and creates a custom property to take off the width
2. add `overflow-x: hidden;` to the body element.

The latter goes against the 'only use classes' and _could_ break pages that use VF1.x components and CSS

The former requires JavaScript.

**Comment by khawkins98** [comment]
Pretty sure you're still tinkering away at this, but shout whenever you're waiting on me for something.

**Comment by sturobson** [comment]
@khawkins98 ready for review now :)

File: components/_previews/_preview--fullhtml.njk [hunk 1]
```diff
@@ -1 +1,28 @@
+<!DOCTYPE html>
```
Comment (khawkins98) [inline components/_previews/_preview--fullhtml.njk]: I think this was an accidental commit?

Comment (sturobson) [reply components/_previews/_preview--fullhtml.njk]: ah, yeah - we don't need that if the issue I had with the GDPR disclaimer and other notification showing up is now fixed. Will remove and push

Comment (sturobson) [reply components/_previews/_preview--fullhtml.njk]: hmm, which bit was the accident? 

I don't think we need preview--nojs which I added now

Comment (khawkins98) [reply components/_previews/_preview--fullhtml.njk]: `_preview--fullhtml` is used by other components, so we shouldn't update that one (well not without considering impacts). -- could you use the default preview, still seems to work ok?

Comment (sturobson) [reply components/_previews/_preview--fullhtml.njk]: I _think_ the only change is whitespace? Will check, fix, and update

Comment (sturobson) [reply components/_previews/_preview--fullhtml.njk]: updated the PR with another push of some codes.

**Comment by khawkins98** [comment]
👍

**Review by khawkins98 — APPROVED** [review]

**Pull request merged**